suites/quincy/rbd/tier-2_rbd_mirror_snapshot_regression.yaml

```diff
@@ -8,183 +8,202 @@
-            verify_cluster_health: true
-            steps:
-              - config:
-                  command: apply
-                  service: rbd-mirror
-                  args:
-                    placement:
-                      label: rbd-mirror
-        ceph-rbd2:
-          config:
-            verify_cluster_health: true
-            steps:
-              - config:
-                  command: apply
-                  service: rbd-mirror
-                  args:
-                    placement:
-                      label: rbd-mirror
-      desc: RBD Mirror daemon deployment using cephadm
-      destroy-clster: false
-      module: test_cephadm.py
-      name: deploy rbd-mirror daemon
+  # - test:
+  #     abort-on-fail: true
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           verify_cluster_health: true
+  #           steps:
+  #             - config:
+  #                 command: apply
+  #                 service: rbd-mirror
+  #                 args:
+  #                   placement:
+  #                     label: rbd-mirror
+  #       ceph-rbd2:
+  #         config:
+  #           verify_cluster_health: true
+  #           steps:
+  #             - config:
+  #                 command: apply
+  #                 service: rbd-mirror
+  #                 args:
+  #                   placement:
+  #                     label: rbd-mirror
+  #     desc: RBD Mirror daemon deployment using cephadm
+  #     destroy-clster: false
+  #     module: test_cephadm.py
+  #     name: deploy rbd-mirror daemon
 
+  # - test:
+  #     name: test_rbd_mirror_snapshot_pool
+  #     module: test_rbd_mirror_snapshot.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           ec_pool_config:
+  #             mirrormode: snapshot
+  #             mode: image
+  #           rep_pool_config:
+  #             mirrormode: snapshot
+  #             mode: image
+  #           snapshot_schedule_level: "pool"
+  #     polarion-id: CEPH-83575375
+  #     desc: Create snapshot based RBD mirrored pools, schedule snapshots at pool level and verify
+  # - test:
+  #     name: test_rbd_mirror_snapshot_cluster
+  #     module: test_rbd_mirror_snapshot.py
+  #     clusters:
+  #       ceph-rbd1:
+  #         config:
+  #           imagesize: 2G
+  #           ec_pool_config:
+  #             mirrormode: snapshot
+  #             mode: image
+  #           rep_pool_config:
+  #             mirrormode: snapshot
+  #             mode: image
+  #           snapshot_schedule_level: "cluster"
+  #     polarion-id: CEPH-83575376
+  #     desc: Create snapshot based RBD mirrored pools, schedule snapshots at cluster level and verify
   - test:
-      name: test_rbd_mirror_snapshot_pool
-      module: test_rbd_mirror_snapshot.py
+      abort-on-fail: True
+      desc: Verify snapshot based mirroring cannot be enabled at pool mode
+      name: Verify snapshot based mirroring cannot be enabled at pool mode
```

Comment:
please provide different statement for test `name` and `desc`
name can be `Test to validate negative scenario with pool based snapshot mirroring`